models/experimental/functional_unet/README.md

```diff
@@ -0,0 +1,20 @@
+## functional_unet Demo
+## How to Run
+
+To run the demo, make sure to build the project, activate the environment, and set the appropriate environment variables.
+For more information, refer [installation and build guide](https://tenstorrent.github.io/tt-metal/latest/get_started/get_started.html#install-and-build).
+
+Use `pytest --disable-warnings models/experimental/functional_unet/demo/demo.py::test_demo` to run all variants of the demo.
+
+If you wish to run the demo with just functional mode `pytest --disable-warnings models/experimental/functional_unet/demo/demo.py::test_demo[False-1-0-32768]`
+
+If you wish to run the demo with perf mode and highest number of batches that fit in L1 (2) `pytest --disable-warnings models/experimental/functional_unet/demo/demo.py::test_demo[True-2-0-32768]`
```

Comment:
Can we change the pytest parametrization so that the filters here are more intuitive? You can just add an `id` for the parameters.

Reply:
Can we also add caveats for which devices are currently supported? i.e. only n150?